src/test/java/spoon/reflect/visitor/CtScannerTest.java

```diff
@@ -207,7 +207,7 @@ public boolean matches(CtInvocation element) {
 
 			// contract: CtScanner only calls methods that have a role and the associated getter
 			if (calledMethods.size() > 0) {
-				problems.add("CtScanner " + visitMethod.getPosition() + " calls unexpected methods: "+calledMethods);
+				fail("CtScanner " + visitMethod.getPosition() + " calls unexpected methods: "+calledMethods);
```

Comment:
why that? A) change all to fail or keep all as `problems.add()`. WDYT?

Reply:
fixed